app/controllers/attendances_controller.rb

```diff
@@ -0,0 +1,46 @@
+require 'awesome_print'
+class AttendancesController < ApplicationController
+  before_action :set_attendance, only: %i[show edit update destroy]
+
+  def index
+    @attendances = Attendance.all
+  end
+
+  def show; end
+
+  def new
+    @attendance = Attendance.new
+  end
+
+  def edit; end
+
+  def create
+    employee = Employee.find_by(private_number: attendance_params[:private_number].to_i)
+    @attendance = Attendance.new(attendance_params)
+    @attendance.employee_id = employee.id
+    return render :new, status: :unprocessable_entity unless @attendance.save
+
+    redirect_to @attendance, notice: 'Attendance was successfully created.'
+  end
+
+  def update
+    return render :edit, status: :unprocessable_entity unless @attendance.update(attendance_params)
+
+    redirect_to @attendance, notice: 'Attendance was successfully updated.'
+  end
+
+  def destroy
+    @attendance.destroy
+    redirect_to attendances_url, notice: 'Attendance was successfully destroyed.'
+  end
```

Comment:
I recommend that you add the logical elimination, instead of the actual elimination.